XRTK-Core/Packages/packages-lock.json

```diff
@@ -62,7 +62,7 @@
     },
     "com.unity.textmeshpro": {
```

Comment:
Are these still needed in the .lock file if they have been removed from the manifest?

Reply:
Lock file is automatically updated by unity and keeps a lost of the direct dependencies in the project.